src/store.js

```diff
@@ -1,10 +1,83 @@
-import Vue from 'vue'
-import Vuex from 'vuex'
+import Vue from 'vue';
+import Vuex from 'vuex';
+import axios from 'axios';
 
-Vue.use(Vuex)
+Vue.use(Vuex);
+const url = 'http://localhost:8081';
```

Comment:
What about adding this to an .env file?

Reply:
I used an env file. Here we can't use the dotenv package as we do on orient-express as this is a server thingy. Webpack is configured to replace the variables VUE_APP_..... that you put on an env file, so I did that instead.

For this to work, I need to adjust now circleci to create this .env file... :| I'm on it